finder-app/finder.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+#filename: finder.sh
+#Author: Pranav Bharadwaj
+#Date: 08/28/2022
+
+#Check if all input arguments are provided
+
+if [ ! $# -eq 2 ]; then
+	echo "Required number of parameters not specified"
```

Comment:
@prvbharadwaj1731 suggestion: help the user execute correctly by providing more details about the parameters required here.